Makefile

```diff
@@ -175,7 +175,7 @@ stop:
 	docker-compose down -v
 
 compile-pot: .state/env/pyvenv.cfg
-	$(BINDIR)/pybabel extract \
+	PYTHONPATH=$(PWD) $(BINDIR)/pybabel extract \
```

Comment:
I was about to send exactly the same patch except also adding this to targets `init-po`, `update-po` and `compile-po`.

Reply:
The extension isn't necessary for those targets at this time.